pre-docs/writing_yaml_plans.md

```diff
@@ -441,7 +441,7 @@ bolt plan convert path/to/my/plan.yaml
 
 This takes the path (relative or absolute) to the YAML plan to be converted and prints the converted Puppet plan to stdout.
 
-**NOTE:** Converting a YAML plan might result in a Puppet plan which is syntactically correct, but behaviorally different from its YAML plan. Always manually verify a converted Puppet plan's functionality. If you convert a YAML plan to puppet and it does *not* have the same behavior as the YAML plan, please [file an issue in our git repo](https://github.com/puppetlabs/bolt/issues).
+**NOTE:** Converting a YAML plan might result in a Puppet plan which is syntactically correct, but behaviorally different from its YAML plan. Always manually verify a converted Puppet plan's functionality. There are some cases where constructs do not translate from YAML plans to Puppet plans which are [listed](#yaml-plan-constructs-which-cannot-be-translated-to-puppet-plans) below. If you convert a YAML plan to puppet and it does *not* have the same behavior as the YAML plan, please [file an issue in our git repo](https://github.com/puppetlabs/bolt/issues).
```

Comment:
capitalization fix: `puppet` to `Puppet` in the sentence that begins `If you convert a YAML plan to puppet...`

Reply:
For clarity, I recommend changing:

>There are some cases where constructs do not translate from YAML plans to Puppet plans which are [listed](#yaml-plan-constructs-which-cannot-be-translated-to-puppet-plans) below.

to:

>There are some constructs that do not translate from YAML plans to Puppet plans. These are [listed](#yaml-plan-constructs-which-cannot-be-translated-to-puppet-plans) below.